src/lib/pay-request/api_utils/ledger.js

```diff
@@ -47,7 +47,47 @@ const ledgerMethods = function ledgerMethods(instance) {
       .then(utilExtractData)
   }
 
-  return { transaction, transactions, events }
+  const getPaymentsByState = function getPaymentsByState(account, fromDate, toDate) {
+    const params = {
+      ...account && { account_id: account },
+      from_date: fromDate,
+      to_date: toDate
+    }
+
+    if (!account) {
+      params.override_account_id_restriction = true
+    }
+
+    const query = Object.keys(params)
+      .map(key => `${key}=${params[key]}`)
+      .join('&')
+
+    return axiosInstance.get(`/v1/report/payments_by_state?${query}`)
+      .then(utilExtractData)
+  }
+
+  const paymentStatistics = function paymentStatistics(account, fromDate, toDate) {
+    const params = {
+      ...account && { account_id: account },
+      ...fromDate && { from_date: fromDate },
+      ...toDate && { to_date: toDate }
+    }
+
+    if (!account) {
+      params.override_account_id_restriction = true
+    }
+
+    const query = Object.keys(params)
+      .map(key => `${key}=${params[key]}`)
```

Comment:
I know this is done elsewhere in `ledger.js` (🤦‍♂ ) but I think Axios will do this for us, I think the equivalent is

```js
return axiosInstance.get(`/v1/report/payments`, { params })
```
(https://github.com/axios/axios#example)